### PR TITLE
delete-and-label

### PR DESCRIPTION
**Pull request by dev-ashishsethi**
https://user-images.githubusercontent.com/89299065/166448976-ea4b2126-e54a-410a-a3ce-32e40fe64624.mp4

image
image

Features:
Delete
Label

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -24,6 +26,8 @@ function App() {
         <Route path="/signUp" element={<SignUp />} />
         {login && <Route path="/home" element={<Home />} />}
         {login && <Route path="/archive" element={<Archive />} />}
+        {login && <Route path="/trash" element={<Delete />} />}
+        {login && <Route path="/label" element={<Label />} />}
       </Routes>
```
Comment (m26unkwn) [inline src/App.js:31]: Try to make protected route provider

File: src/Pages/Label/Label.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+export function Label() {
+  const [labelArr, setLabelArr] = useState([]);
+  const { alllabel, othersData } = useNote();
+  console.log("all label", alllabel);
+  console.log("label array", labelArr);
+  return (
+    <section className="content">
+      {[...alllabel].map((label) => (
+        <LabelCard labels={label} />
```
Comment (m26unkwn) [inline src/Pages/Label/Label.jsx:13]: Like the way you are creating Component

File: src/Components/LabelCard/LabelCard.jsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { useNote } from "../../Context/noteContext";
+import { Card } from "../Cards/Cards";
+
+export function LabelCard({ labels }) {
+  const { othersData } = useNote();
+  let count = 0;
+  const filterData = othersData.filter((note) => note.label.includes(labels));
+  console.log("filtered data on label", filterData);
```
Comment (pankaj-prog) [inline src/Components/LabelCard/LabelCard.jsx:8]: remove console log

File: src/App.js [hunk 4]
```diff
@@ -24,6 +26,8 @@ function App() {
...
       {/* </> */}
```
Comment (pankaj-prog) [inline src/App.js:32]: remove comment

File: src/App.js [hunk 5]
```diff
@@ -10,6 +10,8 @@ import { LandingPage } from "./Pages/LandingPage/LandingPage";
 import { useAuth } from "./Context/loginContext";
 import { SignUp } from "./Components/SignUp/SignUp";
 import { Archive } from "./Pages/Archive/Archive";
+import { Delete } from "./Pages/Delete/Delete";
+import { Label } from "./Pages/Label/Label";
```
Comment (pankaj-prog) [inline src/App.js:14]: suggestion : import all the pages from a single file.

**Review by m26unkwn — APPROVED** [review]

File: src/Pages/Label/Label.jsx [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { useState, useEffect } from "react";
+import { LabelCard } from "../../Components/LabelCard/LabelCard";
+import { useNote } from "../../Context/noteContext";
+
+export function Label() {
+  const [labelArr, setLabelArr] = useState([]);
+  const { alllabel, othersData } = useNote();
+  console.log("all label", alllabel);
+  console.log("label array", labelArr);
```
Comment (pankaj-prog) [inline src/Pages/Label/Label.jsx:9]: remove console logs

**Review by pankaj-prog — APPROVED** [review]

**Pull request merged**